core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -575,8 +575,13 @@ class SparkSubmitSuite
   // NOTE: This is an expensive operation in terms of time (10 seconds+). Use sparingly.
   private def runSparkSubmit(args: Seq[String]): Unit = {
     val sparkHome = sys.props.getOrElse("spark.test.home", fail("spark.test.home is not set!"))
+    val sparkSubmit = if (Utils.isWindows) {
+      Seq(new File("..\\bin\\spark-submit.cmd").getAbsolutePath) ++ args
+    } else {
+      Seq("./bin/spark-submit") ++ args
```

Comment:
OK this PR looks fine to me. My last suggested change would be to simply this changed bit of code to something like ...

```
val sparkSubmitPath = if (Utils.isWindows) "..." else "..."
...
val process = Utils.executeCommand(
   sparkSubmitPath.getAbsolutePath +: args,
...
```
I don't feel strongly about it